src/backstop/backstop_client.ts

```diff
@@ -32,18 +31,16 @@ export interface BackstopClaimArgs {
 }
 
 export interface DrawArgs {
-  from: Address | string;
+  to: Address | string;
   pool_address: Address | string;
   amount: i128;
 }
 
 export class BackstopClient {
```

Comment:
I've been considering the naming of these now that they build operations.

Should we migrate them to `BackstopContract` and so on? They would follow the current `stellar-sdk`'s usage of `Contract` a bit more consistently, since these are just extensions of the Contract object now.

Reply:
Not sure I follow